core/schema/file.go

```diff
@@ -29,7 +29,11 @@ func (s *fileSchema) Install(srv *dagql.Server) {
 		Syncer[*core.File]().
 			Doc(`Force evaluation in the engine.`),
 		dagql.Func("contents", s.contents).
-			Doc(`Retrieves the contents of the file.`),
+			Doc(`Retrieves the contents of the file.`).
+			Args(
+				dagql.Arg("offset").Doc(`Start reading after this line`),
+				dagql.Arg("limit").Doc(`Maximum number of lines to read`),
```

Comment:
I could easily see us adding a limit in terms of bytes in the future, maybe we should name this existing arg `lineLimit` (or `maxLines`, etc.) just to be future proof?

Same idea for `offset`.